validate-bam.wdl

```diff
@@ -54,21 +60,24 @@ workflow ValidateBamsWf {
 task ValidateBAM {
   File bam_file
   String output_basename
-  String validation_mode 
+  String validation_mode
+  String gatk_path
+  
   Int disk_size
   String mem_size
+  String docker
 
   String output_name = "${output_basename}_${validation_mode}.txt"
 
   command {
-    java -Xmx3000m -jar /usr/gitc/picard.jar \
+    ${gatk_path} \
       ValidateSamFile \
-      I=${bam_file} \
-      OUTPUT=${output_name} \
-      MODE=${validation_mode}
+      --INPUT ${bam_file} \
+      --OUTPUT ${output_name} \
+      --MODE ${validation_mode}
   }
   runtime {
-    docker: "broadinstitute/genomes-in-the-cloud:2.2.3-1469027018"
+    docker: docker
     memory: mem_size
     cpu: "1"
     disks: "local-disk " + disk_size + " HDD"
```

Comment:
Note in order to get this WDL to run, I had to [increase the bootDiskSizeGb](https://gatkforums.broadinstitute.org/gatk/discussion/comment/49453#Comment_49453) as the GATK image is currently very large.

Reply:
The latest gatk release (4.0.6.0) has reduced its image size, this should longer be a problem.